evap/rewards/tests/test_tools.py

```diff
@@ -50,3 +51,22 @@ def test_already_got_points(self):
         mommy.make(RewardPointGranting, user_profile=self.student, value=0, semester=self.course.semester)
         self.form.submit()
         self.assertEqual(0, reward_points_of_user(self.student))
+
+class TestGrantRewardPointsParticipationChange(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.course = mommy.make(Course)
+        already_evaluated = mommy.make(Course, semester=cls.course.semester)
+        SemesterActivation.objects.create(semester=cls.course.semester, is_active=True)
+        cls.student = mommy.make(UserProfile, username="student", email="foo@institution.example.com",
```

Comment:
Here, I wasn't sure whether the attributes `student` and `email` are considered user-facing or not. In the test clearly not, but normally, they will be displayed on the page.

Reply:
the maxime is usually to not specify values that you're not testing against, so i would remove them here.

Reply:
But it needs to be an internal user, so the email is required